drivers/SAI.h

```diff
@@ -41,11 +41,19 @@ class SAI : private NonCopyable<SAI> {
         const sai_format_t *fmt = &sai_mode_i2s32, bool is_input = false,
         uint32_t master_clock = 0, bool internal_mclk = false);
 
+    /** Retreive the object status. Allows the ctor to
+     * funnel init errors/warnings back up to the application.
+     */
+    virtual sai_result_t status(void);
```

Comment:
I think it would be better to document arguments/return values in doxygen using @param/@return tags.

Reply:
I have a docs PR coming up after this gets in. I'll include the param/return tags as well in that.